common/version.go

```diff
@@ -64,6 +65,9 @@ func (v Version) ToProto() *pbcommon.NodeVersion {
 
 func (v Version) String() string {
 	pre := ""
+	if BUILDTAGS != "" {
```

Comment:
is that the right condition? Surely you want to check that `BUILDTAGS` contains `insecure`?

Reply:
It's currently only set with insecure. This is not the cleanest way, but we're not likely to add more buildtags anyway and it has to be set manually in the Makefile.